Makefile

```diff
@@ -66,7 +66,7 @@ dev-server: checks
 	@npx concurrently -k \
 	-n "Calypso,Desktop" \
 	"$(MAKE) calypso-dev NODE_ENV=$(NODE_ENV) CALYPSO_ENV=$(CALYPSO_ENV)" \
-	"wait-on http://localhost:3000 && $(MAKE) build-desktop NODE_ENV=$(NODE_ENV)" \
+	"wait-on http://localhost:3000 && $(MAKE) build-desktop-source NODE_ENV=$(NODE_ENV)" \
```

Comment:
Note: Decouple building JavaScript source from the native binaries to enable `make dev-server` on Windows. Native binaries are re-built automatically in a few other places (such as npm post-install script and with `make package`), so this change is fine.